### PR TITLE
Feed page wireframe

### PR DESCRIPTION
**Pull request by GarF1eldGo**
### Summary

- Create feed page wireframe.
- Create rich text editor using [Lexcial](https://lexical.dev/) library.
- Fix docker problem by running `docker compose down`.

### CODE REVIEW

**Comment by juannyG** [comment]
I'm not sure we want to completely disable linting & prettier...

I think you can get them to pass by just running `yarn next:format` before committing changes. I just checked out the branch and doing that got rid of the majority of the linting errors.

**Comment by GarF1eldGo** [comment]
> I'm not sure we want to completely disable linting & prettier...
> 
> I think you can get them to pass by just running `yarn next:format` before committing changes. I just checked out the branch and doing that got rid of the majority of the linting errors.

This command is helpful! I will try to solve the linting errors this afternoon.

File: packages/nextjs/app/feed/page.tsx [hunk 1]
```diff
@@ -0,0 +1,28 @@
+import ContentPage from "./contentPage";
+import FeedHeaderPage from "./header";
+import LeftPage from "./leftPage";
+import RightPage from "./rightPage";
+import type { NextPage } from "next";
+
+const FeedPage: NextPage = () => {
```
Comment (juannyG) [inline packages/nextjs/app/feed/page.tsx:7]: Might be worth adding a TODO here: If the user is not registered, we redirect them back to `/`

Example: https://github.com/juannyG/coms6998/blob/dev/packages/nextjs/app/home/page.tsx#L29-L38

(Would love your thoughts on if there's a way to add this check without having to put it on _every `page.tsx`_)

Comment (GarF1eldGo) [reply packages/nextjs/app/feed/page.tsx:7]: I have an idea about the Single Sign On (SSO) feature. I could implement some protective functions in a .tsx file that manages the root page. It seems I could modify app/layout.tsx to incorporate redirection logic.https://stackoverflow.com/questions/78319443/how-to-do-azure-single-sign-on-with-next-js-14-and-app-router

**Review by juannyG — APPROVED** [review]

**Pull request merged**